### PR TITLE
Add handling in the CheckMainFeatureForCSharpLanguageTest for possible failing after first initialization

### PR DESCRIPTION
**Pull request by musienko-maxim**
### What does this PR do?
Sometimes for first LS. initialization the default timeout is not enough. For this - in the test has been added handle that invokes command for getting dependencies by terminal command and checks main of LS. features again

### What issues does this PR fix or reference?
https://github.com/eclipse/che/issues/7952
@tolusha @dmytro-ndp @vparfonov

### CODE REVIEW

File: selenium/che-selenium-test/src/test/java/org/eclipse/che/selenium/languageserver/CheckMainFeatureForCSharpLanguageTest.java [hunk 1]
```diff
@@ -75,4 +93,35 @@ public void checkLaunchingCodeserver() throws Exception {
...
+        new WebDriverWait(seleniumWebDriver, MINIMUM_SEC)
+            .until(
+                ExpectedConditions.presenceOfAllElementsLocatedBy(
+                    By.xpath(xpathLocatorForEventMessages)));
+    StringBuilder allMessagesBuilder = new StringBuilder();
+    textMessages.forEach(message -> allMessagesBuilder.append(message.getAttribute("textContent")));
+    String agrigatedMessages = allMessagesBuilder.toString();
+    return (isNullOrEmpty(agrigatedMessages))
+        ? false
+        : (agrigatedMessages.contains("Timeout initializing error"));
```
Comment (dmytro-ndp) [inline selenium/che-selenium-test/src/test/java/org/eclipse/che/selenium/languageserver/CheckMainFeatureForCSharpLanguageTest.java]: `agrigated` > `aggregated`
Please, check the grammar before committing.

**Review by tolusha — APPROVED** [review]

**Review by vparfonov — APPROVED** [review]

File: selenium/che-selenium-test/src/test/java/org/eclipse/che/selenium/languageserver/CheckMainFeatureForCSharpLanguageTest.java [hunk 2]
```diff
@@ -75,4 +93,35 @@ public void checkLaunchingCodeserver() throws Exception {
...
+
+  private boolean isInitLanguageServerFail() {
+    String xpathLocatorForEventMessages =
+        "//div[contains(@id,'gwt-debug-notification-wrappergwt-uid')]";
+    List<WebElement> textMessages =
+        new WebDriverWait(seleniumWebDriver, MINIMUM_SEC)
+            .until(
+                ExpectedConditions.presenceOfAllElementsLocatedBy(
+                    By.xpath(xpathLocatorForEventMessages)));
+    StringBuilder allMessagesBuilder = new StringBuilder();
```
Comment (dmytro-ndp) [inline selenium/che-selenium-test/src/test/java/org/eclipse/che/selenium/languageserver/CheckMainFeatureForCSharpLanguageTest.java]: Sorry for late comment. But this part can be simplified a bit:
```javascript
    return textMessages.stream().anyMatch(
        message -> message.getAttribute("textContent")
            .contains("Timeout initializing error"));
```

File: selenium/che-selenium-test/src/test/java/org/eclipse/che/selenium/languageserver/CheckMainFeatureForCSharpLanguageTest.java [hunk 3]
```diff
@@ -75,4 +93,35 @@ public void checkLaunchingCodeserver() throws Exception {
...
+  private void reInitLanguageServer() {
+    commandsPalette.openCommandPalette();
+    commandsPalette.startCommandByDoubleClick(COMMAND_NAME_FOR_RESTORE_LS);
+    consoles.waitExpectedTextIntoConsole("Restore completed");
+    editor.closeAllTabs();
+    projectExplorer.openItemByPath(PROJECT_NAME + "/Program.cs");
+    loader.waitOnClosed();
+  }
+
+  private boolean isInitLanguageServerFail() {
```
Comment (dmytro-ndp) [inline selenium/che-selenium-test/src/test/java/org/eclipse/che/selenium/languageserver/CheckMainFeatureForCSharpLanguageTest.java]: `isInitLanguageServerFail` > `isLanguageServerInitFailed`

File: selenium/che-selenium-test/src/test/java/org/eclipse/che/selenium/languageserver/CheckMainFeatureForCSharpLanguageTest.java [hunk 4]
```diff
@@ -75,4 +93,35 @@ public void checkLaunchingCodeserver() throws Exception {
     editor.typeTextIntoEditor(";");
     editor.waitAllMarkersDisappear(ERROR_MARKER);
   }
+
+  private void checkInitStateAndLaunchLanguageServer() {
```
Comment (dmytro-ndp) [inline selenium/che-selenium-test/src/test/java/org/eclipse/che/selenium/languageserver/CheckMainFeatureForCSharpLanguageTest.java]: `checkInitStateAndLaunchLanguageServer` > `ensureLanguageServerInitialized`

**Review by dmytro-ndp — APPROVED** [review]

**Pull request merged**